docs/source/_ecf_usage_participant_actions.html.erb

```diff
@@ -324,3 +323,59 @@ takes precedence. In this instance, the provider should not change the mentor’
 </pre>
 <p class="govuk-body-m">This will return an <a href="/api-reference/reference-v1#schema-ecfparticipantresponse" class="govuk-link">ECF participant record</a> with the updates to the record included.</p>
 <p class="govuk-body-m">See <a href="/api-reference/reference-v1#api-v1-participants-ecf-id-withdraw-put" class="govuk-link">withdraw ECF participant</a> endpoint.</p>
+
+<h2 id="declaring-that-an-ecf-participant-has-started-their-course" class="govuk-heading-l">Declaring that an ECF participant has started their course</h2>
+<p class="govuk-body-m">This scenario begins after it has been confirmed that an ECF participant is ready to begin their induction training.</p>
+
+<h3 class="govuk-heading-m" id="provider-confirms-ECF-participant-has-started">1. Provider confirms an ECF participant has started</h3>
+<p class="govuk-body-m">Confirm an ECF participant has started their induction training before Milestone 1.</p>
+<p class="govuk-body-m"><pre><code>POST /api/v1/participant-declarations</code></pre></p>
+
+<p class="govuk-body-m">With a <a href="/api-reference/reference-v1#schema-ecfparticipantstarteddeclaration" class="govuk-link">request body containing an ECF participant declaration</a>.</p>
+<p class="govuk-body-m">This returns <a href="/api-reference/reference-v1#schema-singleparticipantdeclarationresponse" class="govuk-link">participant declaration</a>.</p>
+
+<p class="govuk-body-m">This endpoint is idempotent - submitting exact copy of a request will return the same response body as submitting it the first time.</p>
+<p class="govuk-body-m">See <a href="/api-reference/reference-v1#api-v1-participant-declarations-post" class="govuk-link">confirm ECF participant declarations</a> endpoint.</p>
+
+<h3 class="govuk-heading-m" id="provider-records-ECF-participant-declaration-id">2. Provider records the ECF participant declaration ID</h3>
+<p class="govuk-body-m">Store the returned ECF participant declaration ID for future management tasks.</p>
+
+<h2 id="declaring-that-an-ecf-participant-has-retained-their-course" class="govuk-heading-l">Declaring that an ECF participant has reached a retained milestone</h2>
+<p class="govuk-body-m">This scenario begins after it has been confirmed that an ECF participant has completed enough of their course to meet a milestone.</p>
+
+<h3 class="govuk-heading-m" id="provider-confirms-ECF-participant-has-been-retained">1. Provider confirms an ECF participant has been retained</h3>
+<p class="govuk-body-m">1. Confirm an ECF participant has been retained by the appropriate number of months for this retained event on their ECF course.</p>
```

Comment:
Looks like there are two `1.` prefixes here in a row.

```suggestion
<h3 class="govuk-heading-m" id="provider-confirms-ECF-participant-has-been-retained">1. Provider confirms an ECF participant has been retained</h3>
<p class="govuk-body-m">Confirm an ECF participant has been retained by the appropriate number of months for this retained event on their ECF course.</p>
```

Reply:
All fixed. Tks @peteryates